docs/eventlisteners.md

```diff
@@ -430,618 +384,215 @@ metadata:
-the "Sink".  If you happen to have a HTTP Firewall between your Cluster and external entities, that means more
-administrative cost, opening ports in the firewall for each Service, unless you can employ Kubernetes `Ingress` to
-serve as a routing abstraction layer for your set of EventListeners. 
-
-### Multiple EventListeners (Multiple EventListeners per Namespace)
-
-Multiple EventListeners per namespace will most likely mean more ServiceAccount/Secret/RBAC manipulation for
-the administrator, as some of the built in generation of those artifacts as part of namespace creation are not
-applicable.
-
-However you will save some on the `etcd` storage costs by reducing the number of namespaces.
-
-Multiple EventListeners and potential Firewall concerns still apply (again unless you employ `Ingress`).
-
-### ServiceAccount per EventListenerTrigger
-
-Being able to set a ServiceAccount on an EventListenerTrigger allows for finer grained permissions as well.
-
-You still have to create the additional ServiceAccounts.
-
-But staying within 1 namespace and minimizing the number of EventListeners with their associated "Sinks" minimizes 
-concerns around `etcd` storage and port considerations with Firewalls if `Ingress` is not utilized.
-
----
-
-Except as otherwise noted, the content of this page is licensed under the
-[Creative Commons Attribution 4.0 License](https://creativecommons.org/licenses/by/4.0/),
-and code samples are licensed under the
-[Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
+You will get a response similar to the following:
+```
+NAME                       ADDRESS                                                             AVAILABLE   REASON                     READY   REASON
+tls-listener-interceptor   http://el-tls-listener-interceptor.default.svc.cluster.local        True        MinimumReplicasAvailable
+```
 
-## EventListener Secure Connection
+Where for each returned line, the column values are, from left to right:
+- `NAME` - name of the `EventListener`
+- `ADDRESS` - IP address or URL of the `EventListener`
+- `AVAILABLE` - readiness state of the associated `Deployment` and `Service`
+- `REASON` - reason for the value displayed in the `AVAILABLE` column
+- `READY` - readiness state of the Kubernetes Custom Resource object specified in the `EventListener`
+- `REASON` - reason for the value displayed in the `READY` column
 
-Triggers now support both `HTTP` and `HTTPS` connection by adding few configuration to eventlistener.
+**Note:** The status messaging described above is being refactored. For more information, see [Issue 932](https://github.com/tektoncd/triggers/issues/932).
 
-To setup TLS connection add two set of reserved environment variables `TLS_CERT` and `TLS_KEY` using `secretKeyRef` env type 
-where we need to specify the `secret` which contains `cert` and `key` files. See the full [example](../examples/eventlistener-tls-connection/README.md) for more details.
+## Using `EventListener` logging
```

Comment:
Specifying EventListener logging options?

Reply:
Or Configuring Logging for EventListener?

Reply:
Reworded to "Configuring logging for `EventListeners`".